File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.java
class Solution {
    public boolean arrayStringsAreEqual(String[] word1, String[] word2) {
        String temp1  = "";
        String temp2 = "";
        
        for(int i = 0; i < word1.length; i++){
            temp1+=word1[i];
        }
        for (int j = 0; j < word2.length; j++){
            temp2 += word2[j];
        }
        return temp1.equals(temp2);
    }
}

File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.py
class Solution:
    def arrayStringsAreEqual(self, word1: List[str], word2: List[str]) -> bool:
        temp = ""
        temp2 = ""
        for i in word1:
            temp+=i
            
        for j in word2:
            temp2+=j
        
        if temp == temp2:
            return True
        else:
            return False;